R/set_derived_variable_labels.R

```diff
@@ -1,54 +1,69 @@
 #' Apply variable labels to data frame
 #'
-#' Takes labels from the Derived Variables excel file and applies them to the
+#' Takes labels from the Derived Variables CSV file and applies them to the
 #' passed data frame.
-#' The excel sheet must have columns `"varname"` and `"label"`.
+#' The excel sheet must have columns `"df_name"`, `"df_label"`, `"var_name"`, and `"var_label"`.
+#' The returned data frame is also labelled with the value in `"df_label"`.
 #'
 #' @param data Data frame
-#' @param path Path to Derived Variables xls/xlsx file
-#' @param drop Logical indicating whether to drop unlabelled variables
-#' @inheritParams readxl::read_excel
+#' @param path Path to CSV file
+#' @param df_name string indicating the name of the data frame to apply labels to
```

Comment:
would the `data` and `df_name` arguments ever be different, or would you always expect them to be the same?

Reply:
I think they should always be the same, and we could pretty easily grab the name from the function call if the user doesn't use any pipe or uses the base R pipe. Otherwise, it'll come in as a `.`, i think. I'll add something as default and we'll see how it goes 🤞🏼

Reply:
updated pushed!

I also added more checks on the df_name, because the new default value can be a little strange depending on how the data frame is passed to the function